cf_units/__init__.py

```diff
@@ -1509,15 +1513,16 @@ def _op_common(self, other, op_func):
             raise ValueError("Cannot %s a 'no-unit'." % op_label)
 
         if self.is_unknown() or other.is_unknown():
-            result = _Unit(_CATEGORY_UNKNOWN, None)
+            result = Unit(_UNKNOWN_UNIT_STRING)
```

Comment:
This is the alternative to #125. Really, let's avoid encoding the meaning of UNKNOWN unit in multiple places, and let the constructor (``__init__``) deal with it.